### PR TITLE
Update SpotPass dumping docs to mention BOSS task databases

### PR DESCRIPTION
**Pull request by jonbarrow**
Updates the network dumping docs SpotPass section to include information on how to submit BOSS task databases.

These databases are invaluable, as they give us information on tons of BOSS tasks without the need for network dumps. These databases contain the BOSS application IDs and associated task names, but no actual BOSS content. For actual content, an archiver or real HTTP dumps is still needed

Relies on the pending Bandwidth PR

### CODE REVIEW

**Review by DaniElectra — COMMENTED** [review]
Also, we should use numbering and not dashes and finish the sentences with dots for consistency with the other guides on this page?

File: docs/en_US/network-dumps.md [hunk 1]
```diff
@@ -109,15 +109,43 @@ Once you have the proxy server running and your console connected to it, use the
...
+
+We encourage everyone to upload their task databases, even if you have not been online in a long time. These databases may contain SpotPass information from games you have played in the past.
+
+## Wii U
+The Wii U stores a separate database of BOSS tasks per user. Each one must be dumped and submitted individually.
+
+1. Connect to the Wii U using [FTP](https://wiki.hacks.guide/wiki/Wii_U:FTP#Aroma-0)
+2. Navigate to `/storage_mlc/usr/save/system/boss`
+3. Copy all the folders in this directory to your PC
+4. Submit the `task.db` file in each folder using Bandwidth
```
Comment (DaniElectra) [inline docs/en_US/network-dumps.md:134]:
```suggestion
1. Connect to the Wii U using [FTP](https://wiki.hacks.guide/wiki/Wii_U:FTP#Aroma-0).
2. Navigate to `/storage_mlc/usr/save/system/boss`.
3. Copy all the folders in this directory to your PC.
4. Submit the `task.db` file in each folder using Bandwidth.
```

File: docs/en_US/network-dumps.md [hunk 2]
```diff
@@ -109,15 +109,43 @@ Once you have the proxy server running and your console connected to it, use the
...
+1. Launch [GodMode9](https://github.com/d0k3/GodMode9)
+2. Navigate to `SYSNAND CTRNAND > data > longstring > sysdata > 00010034`
+3. Select `00000000`. If your file is not named `00000000` you may still continue, though we cannot guarantee this is the correct file. If you have more than one file, repeat the following steps for each
+4. Select `Mount as DISA image`
+5. Press `A` to mount and enter the image
+6. Select `partitionA.bin`. If your file is not named `partitionA.bin` you may still continue, though we cannot guarantee this is the correct file. If you have more than one file, repeat the following steps for each
+7. Select `Copy to 0:/gm9/out`
+8. Turn off your console and eject the SD card
+9. Open your SD card on your computer
+10. Submit the partition BIN file using Bandwidth
```
Comment (DaniElectra) [inline docs/en_US/network-dumps.md:148]:
```suggestion
1. Launch [GodMode9](https://github.com/d0k3/GodMode9).
2. Navigate to `SYSNAND CTRNAND > data > longstring > sysdata > 00010034`.
3. Select `00000000`. If your file is not named `00000000` you may still continue, though we cannot guarantee this is the correct file. If you have more than one file, repeat the following steps for each.
4. Select `Mount as DISA image`.
5. Press `A` to mount and enter the image.
6. Select `partitionA.bin`. If your file is not named `partitionA.bin` you may still continue, though we cannot guarantee this is the correct file. If you have more than one file, repeat the following steps for each.
7. Select `Copy to 0:/gm9/out`.
8. Turn off your console and eject the SD card.
9. Open your SD card on your computer.
10. Submit the partition BIN file using Bandwidth.
```

**Review by DaniElectra — COMMENTED** [review]
mostly for consistency

**Review by DaniElectra — APPROVED** [review]
looks good

**Comment by krisutofu** [comment]
Sorry if this is not the right place. I have reading the docs and am still clueless how to dump these BOSS task databases by myself. I have many red games on the list. Could you add more information? Otherwise, I'll try the docker way but I still don't know what I need to do on the Wii U to trigger the HTTP packets.

**Pull request merged**